elftools/dwarf/callframe.py

```diff
@@ -277,17 +277,19 @@ def _parse_cie_augmentation(self, header, entry_structs):
             'Unhandled augmentation string: {}'.format(repr(augmentation)))
 
         available_fields = {
-            b'z': entry_structs.Dwarf_uleb128('length'),
-            b'L': entry_structs.Dwarf_uint8('LSDA_encoding'),
-            b'R': entry_structs.Dwarf_uint8('FDE_encoding'),
+            b'z': 'length' / entry_structs.Dwarf_uleb128,
+            b'L': 'LSDA_encoding' / entry_structs.Dwarf_uint8,
+            b'R': 'FDE_encoding' / entry_structs.Dwarf_uint8,
             b'S': True,
-            b'P': Struct(
-                'personality',
-                entry_structs.Dwarf_uint8('encoding'),
-                Switch('function', lambda ctx: ctx.encoding & 0x0f, {
-                    enc: fld_cons('function')
+            b'P': 'personality' / Struct(
```

Comment:
Can you fix the formatting here a bit? It's hard to see that the following fields are inside the Struct
Maybe indent these parts more, or extract them into a separate var before `available_fields`?